internal/driver/testdata/testflame.js

```diff
@@ -0,0 +1,86 @@
+(function () {
```

Comment:
nit: I would maybe give the function a name and then call it explicitly at the end. This syntax with parens gave me a bit of a pause when reading the code.